cb-talent-development/employer-style-base
# CE-216: Refresh Email Signup Footer (ESB)

**Purpose**
> As a site visitor, when I scroll to the bottom of any page, I want to see a newsletter subscription bar rise up from the footer to catch my eye and prompt me to sign up for some tasty content.

This PR adds a new button to be used for signup forms called `signup-button`.

**JIRA**
https://careerbuilder.atlassian.net/browse/CE-216

**Changes**
* Improvements and fixes
  * New `signup-button`

* Changes to developer setup/environment
  * N/A

* Architectural changes
  * N/A

* Migrations or Steps to Take on Production
  * N/A
  
* Library changes
  * N/A

* Side effects
  * N/A

**Screenshots**
* Before
![screen shot 2017-10-17 at 5 10 47 pm](https://user-images.githubusercontent.com/5348098/31692151-407c7514-b35e-11e7-9db3-a1379accfd06.png)

![screen shot 2017-10-17 at 5 11 01 pm](https://user-images.githubusercontent.com/5348098/31692155-4359b40e-b35e-11e7-9363-f4c6cad860c4.png)

* After
![screen shot 2017-10-17 at 5 11 13 pm](https://user-images.githubusercontent.com/5348098/31692162-49be9292-b35e-11e7-962e-7ba22c1d5a2c.png)

![screen shot 2017-10-17 at 5 11 22 pm](https://user-images.githubusercontent.com/5348098/31692168-4f7701b0-b35e-11e7-83bc-c6e34820b4f7.png)

**Feature Server**
http://web.employer-1.development.c66.me/

**How to Verify These Changes**
* Specific pages to visit
  * All pages

* Steps to take
  * Go to any page on the site and scroll to the bottom to see the new signup form
  * Compare it to the mocks for both mobile and desktop
  * View it on your phone to see if it looks ok
  * Resize your browser
  * Compare the button to the hiringDot site. It should behave similarly when resizing the browser.

* Responsive considerations
  * Resize your browser and test it on your phone. The three column layout should collapse to a stacked view around 750px.

**Relevant PRs/Dependencies**
  * https://github.com/cbdr/employer/pull/1016

**Additional Information**
N/A
